### ISSUE TITLE
Panic: unknown field "max_retries_to_fulfil_nonces_reputer" in types.Params On Starting Chain

### ISSUE THREAD

**Issue by Muneeb147** [allora-network/allora-chain#674]
## Issue:
I'm following the instructions for "Running A Full Node" as Validator from this documentation:
https://docs.allora.network/devs/validators/run-full-node

The chain is not started and following panic error is thrown from golang cosmos-sdk:

```
panic: unknown field "max_retries_to_fulfil_nonces_reputer" in types.Params
```

### What Configuration I'm using?

As per given instructions, I'm using your latest github release (https://github.com/allora-network/allora-chain/releases/tag/v0.6.1)

**Docker Image**: `alloranetwork/allora-chain:v0.5.0-docker-upgrade` (It is already used in docker-compose)

I even tried with another docker image (`alloranetwork/allora-chain:v0.6.1`) as well BUT same error.

Please help and guide how we can run a full node so that it may eventually catchup the chain and we can start as validator.

<img width="1195" alt="Screenshot 2024-11-17 at 12 10 07 AM" src="https://github.com/user-attachments/assets/ba92c682-f8f0-42e3-aa33-b211e3672b0b">

**Comment by relyt29** [comment]
this is _probably_ related to the structure of the genesis file and whether you're using the right genesis

@kush-alloralabs if you have time can you try and reproduce?

**Comment by Muneeb147** [comment]
> this is _probably_ related to the structure of the genesis file and whether you're using the right genesis
> 
> @kush-alloralabs if you have time can you try and reproduce?

So I haven't modify genesis file or any other chain configuration. I am just using docker image and launching a container to start a chain with a full node (as per the instructions given here https://docs.allora.network/devs/validators/run-full-node)

**Comment by BONESDAO** [comment]
I also have this question. when I use `docker compose up` the terminal will report 

```
panic: unknown field "max_retries_to_fulfil_nonces_reputer" in types.Params
```` 

I was use the version is allora-chain release-v0.6.6, and this error will lead to the error 

```
sample_validator  | Error: exit status 2
sample_validator  | 5:45PM ERR error="exit status 2" module=cosmovisor
```

**Comment by relyt29** [comment]
Yeah this has to do with the genesis.json having parameters that were deleted in later versions. When you sync from genesis, you have to basically have a copy of every binary from genesis, so allora version v0.1 all the way to now v0.6.6, and then basically upgrade the binary to get to the current version

I believe the docker images probably should be updated to have all these images and properly sync from genesis, but in the meantime, sync from a snapshot, which will sidestep this issue and load you in with a relatively recent state
